src/validate-spec.js

```diff
@@ -197,370 +198,686 @@ describe('validate', () => {
       ],
     };
 
-    spyOn(JsonLoaderHelper, 'getFile').and.callFake(url => ({
-      errorCode: JsonLoaderHelper.ERROR_NONE,
-      statusCode: 200,
-      data: activityList,
-      url,
-      exception: null,
-      contentType: 'application/json',
-      fetchTime: (new Date()).valueOf(),
-    }));
+    nock.disableNetConnect();
+    nock('https://openactive.io')
+      .get('/activity-list')
+      .reply(200, activityList, {
+        'Content-Type': 'application/ld+json',
+      });
 
     options = new OptionsHelper({
       loadRemoteJson: true,
       version: 'latest',
     });
   });
+  afterEach(() => {
+    nock.cleanAll();
+    nock.enableNetConnect();
+  });
 
-  it('should return a failure if passed an invalid model', () => {
-    const data = {};
+  describe('validateAsync()', () => {
```

Comment:
There is also a `describe('validateSync()', () => {` describe block which is a duplicate of this block except that it tests `validateSync` instead